src/test/java/com/bobocode/bibernate/integration/DateTypeConverterIntegrationTest.java

```diff
@@ -0,0 +1,153 @@
+package com.bobocode.bibernate.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.bobocode.bibernate.integration.entity.DateTestEntity;
+import com.bobocode.bibernate.session.Session;
+import com.bobocode.bibernate.session.SessionFactoryImpl;
+import com.bobocode.parser.YamlPropertyParser;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.concurrent.ThreadLocalRandom;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+class DateTypeConverterIntegrationTest {
+
+    private Session session;
+    private SessionFactoryImpl sessionFactory;
+
+    @BeforeEach
+    void openSession() {
+        YamlPropertyParser parser = new YamlPropertyParser();
+        sessionFactory = new SessionFactoryImpl(parser, "date-converter-integration-test");
+        session = sessionFactory.openSession();
+    }
+
+    @AfterEach
+    void tearDown(TestInfo testInfo) {
+        if (testInfo.getTags().contains("SkipCleanup")) {
+            return;
+        }
+        session.close();
+        sessionFactory.close();
+    }
+
+    @Test
```

Comment:
Generally, we would use `@DisplayName` instead of underscores, but I don't mind

Reply:
I used underscores here to show difference between such methods as localDateConverter_save() and localDateConverter_find(). they have the same base but tests different operations.